framework/graphics/vulkan_check_buffer_references.h

```diff
@@ -0,0 +1,35 @@
+//
+// Created by crocdialer on 31.07.24.
```

Comment:
This file is missing the usual copyright header

Reply:
right, will fix that.

Reply:
https://github.com/LunarG/gfxreconstruct/pull/1648 - thanks @panos-lunarg!